### PR TITLE
Replace PIL and StringIO by OpenCV

### PR DESCRIPTION
**Pull request by LiuXiaolong19920720**
My platform is Windows 7 with Python 2.7. There is a bug about StringIO I can not fix, so I replace the StringIO and PIL module by OpenCV. Delete defined resize() function and replace it by cv2.resize(). And use cv2.imread() to read images. It then work well.

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by realzheng** [comment]
But the output score is different.

File: classify_nsfw.py [hunk 1]
```diff
@@ -4,38 +4,16 @@
...
-    img_data = str(data)
-    im = Image.open(StringIO(img_data))
-    if im.mode != "RGB":
-        im = im.convert('RGB')
-    imr = im.resize(sz, resample=Image.BILINEAR)
-    fh_im = StringIO()
-    imr.save(fh_im, format='JPEG')
-    fh_im.seek(0)
-    return bytearray(fh_im.read())
+import cv2
```
Comment (varenc) [inline classify_nsfw.py]: this seems like a massive dependency to require when all you're trying to do is get around some StringIO bug and resize an image.

Comment (varenc) [inline classify_nsfw.py]: personallyI wouldn't want to install opencv just to run this!  There's probably some other sort of workaround for the StringIO issue.  Can you describe what it is specifically? it's probably fixable just by tweaking the usage here.

Comment (LiuXiaolong19920720) [inline classify_nsfw.py]: nsfw_error
This is the bug and I am unfamiliar with the StringIO.By the way,I have used OpenCV for some time,that's why OpenCV come to my mind firstly.

Comment (chrme) [inline classify_nsfw.py]: just alter the line
```python
image_data = open(args.input_file).read()
```

with
```python
image_data = open(args.input_file, 'rb').read()
```

File: classify_nsfw.py [hunk 2]
```diff
@@ -27,16 +27,18 @@ def resize_image(data, sz=(256, 256)):
     :returns bytearray:
         A byte array with the resized image
     """
-    img_data = str(data)
```
Comment (kulkarnivishal) [inline classify_nsfw.py:30]: what's the format of data

**Pull request closed without merging**